### ISSUE TITLE
Not on PYPI?

### ISSUE THREAD

**Issue by danmichaelo** [svpino/rfeed#9]
Hi. This looks like a nice library, but I couldn't `pip install rfeed`. Publishing on PyPI is really easy, see https://python-packaging-user-guide.readthedocs.io/en/latest/distributing/

**Comment by egorsmkv** [comment]
I'm published this package on PyPI: https://pypi.python.org/pypi/rfeed with changes. see: https://github.com/egorsmkv/rfeed

**Comment by rafaelbco** [comment]
What is the "official" repository for this project? @egorsmkv or @svpino ?

Looks like @egorsmkv is doing the releases on PyPI. I'd like to fill an issue and probably a pull request afterwards. @egorsmkv repository does not accept issues though.

Can you please help me?

**Comment by egorsmkv** [comment]
@rafaelbco official repo is @svpino https://github.com/svpino/rfeed

@rafaelbco I'm opened issues in my repo.

**Comment by rafaelbco** [comment]
So, it seems @svpino  is the official repo but I'm better off opening issues and PRs ate @egorsmkv because PyPI releases are made based on that repo. Am I right?

**Comment by egorsmkv** [comment]
@rafaelbco Yes, you are right.

**Comment by nowster** [comment]
Since @egorsmkv has archived his fork, and development has been happening here again, perhaps PyPI should be using this repo instead?

**Comment by devnoname120** [comment]
Just do this:

```shell
pip install --upgrade https://github.com/svpino/rfeed/tarball/master
```

**Comment by drf5n** [comment]
Yes, the https://github.com/egorsmkv/rfeed fork was last modified on Jul 4, 2016 and is now archived with this message:

> This repository was archived by the owner on Jun 1, 2022. It is now read-only.

There are other forks listed in this repository's Insights/Network tab at https://github.com/svpino/rfeed/network

* https://github.com/jay-tuckey/rfeed -- updated packaging and python
* https://github.com/johnjreiser/rfeed -- GeoRSS
